frontend/pages/Parts/[deviceHandle]/[itemTypeHandle].tsx

```diff
@@ -48,16 +48,11 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
          getGlobalSettings(),
          getStoreList(),
          getStoreByCode('us'),
-         findProductList(
-            {
-               deviceTitle: {
-                  eq: deviceTitle,
-               },
+         findProductList({
```

Comment:
Is the Algolia search state loaded by the time this code runs? If so, we could use the hook to get the item type, and pass it to findProductList()